lacommunaute/stats/admin.py

```diff
@@ -1,16 +1,44 @@
+from dateutil.relativedelta import relativedelta
 from django.contrib import admin
 
+from lacommunaute.forum.models import Forum
 from lacommunaute.stats.models import ForumStat, Stat
 
 
+class ForumWithStatsFilter(admin.SimpleListFilter):
+    title = "Forum"
+    parameter_name = "forum"
+
+    def lookups(self, request, model_admin):
+        forums_with_stats = model_admin.model.objects.values_list("forum", flat=True).distinct()
```

Comment:
Pas grave s’il y a des doublons dans le `__in`.
```suggestion
        forums_with_stats = model_admin.model.objects.values_list("forum", flat=True)
```